lib/vanagon/platform.rb

```diff
@@ -435,6 +435,23 @@ def add_build_repository(*args)
       end
     end
 
+    def generate_compiled_archive(project)
```

Comment:
probably want yardoc for this method, since we have 'em for the rest.

Reply:
ok, will add